### PR TITLE
Update kube dashboard in docs and addons/

### LINKED ISSUE #1375: Update link in addons doc
https://github.com/kubernetes/kops/blob/master/docs/addons.md#dashboard

Seems to be pointing to old version.  Also make sure we're running the latest dashboard.

### PR DESCRIPTION
**Pull request by geojaz**
Closes #1375

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1379)

### CODE REVIEW

File: docs/addons.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The [dashboard project](https://github.com/kubernetes/dashboard) provides a nice
 
 Install using:
 ```
-kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/kubernetes-dashboard/v1.4.0.yaml
+kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
```
Comment (geojaz) [inline docs/addons.md]: Is it better to link to this project so we always have the latest? Or to the versioned manifest that I copied over?

**Comment by kris-nova** [comment]
@k8s-bot ok to test

File: docs/addons.md (hunk 1 above)
Comment (chrislovecnm) [inline docs/addons.md]: We would prefer to vendor

Comment (geojaz) [inline docs/addons.md]: Ok this should be good to go!

**Comment by justinsb** [comment]
Where did you get this manifest from?  It looks a little different to the one we're shipping for 1.4.0...

**Comment by justinsb** [comment]
Aha - I think we want to update to this one: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dashboard/dashboard-controller.yaml

AFAICT, we want to copy _our_ 1.4.0 manifest (so you get the k8s-addon label), and then bump the image version.  You probably want to change the deployment name also `kubernetes-dashboard-v1.4.0 -> kubernetes-dashboard`.  And the version tag.

Upstream doesn't have a version tag, but I think it should.

We add the k8s-addon label so that we can pass this to kubectl prune to remove "extra" resources.  I'm not sure off-the-top-of-my-head whether this has landed yet in kubectl, but it certainly isn't yet wired up in channels.  But soon :-)

**Pull request merged**